srcpkgs/python3-jaraco.functools/template

```diff
@@ -1,21 +1,26 @@
 # Template file for 'python3-jaraco.functools'
 pkgname=python3-jaraco.functools
-version=3.0.0
+version=3.0.1
 revision=1
 archs=noarch
 wrksrc="jaraco.functools-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-more-itertools python3-jaraco"
+checkdepends="${depends} python3-pytest python3-jaraco.classes"
 short_desc="Functools like those found in stdlib (Python3)"
 maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="MIT"
 homepage="https://github.com/jaraco/jaraco.functools"
 changelog="https://raw.githubusercontent.com/jaraco/jaraco.functools/master/CHANGES.rst"
 distfiles="${PYPI_SITE}/j/jaraco.functools/jaraco.functools-${version}.tar.gz"
-checksum=5cb0eea0f254584241c519641328a4d4ec2001a86c3cd6d17a8fd228493f6d97
+checksum=9fedc4be3117512ca3e03e1b2ffa7a6a6ffa589bfb7d02bfb324e55d493b94f4
+
+do_check() {
+	pytest -vv --override-ini="addopts=--doctest-modules"
+}
 
 post_install() {
-	rm -r ${DESTDIR}/${py3_sitelib}/jaraco/__init__.py
+	rm -f ${DESTDIR}/${py3_sitelib}/jaraco/__init__.py
```

Comment:
Nice catch, but -f isn't necessary either.

Reply:
Ok, removed it. I had added it just in case